src/server/cluster/cluster_family.h

```diff
@@ -62,7 +62,8 @@ class ClusterFamily {
   void DflyClusterFlushSlots(CmdArgList args, ConnectionContext* cntx);
 
  private:  // Slots migration section
-  void DflySlotMigrationStatus(CmdArgList args, ConnectionContext* cntx);
```

Comment:
There are a lot of warnings + actually cases where we forget to lock. I will fix them in a separate PR this is already getting big